src/osd/OSD.cc

```diff
@@ -10295,6 +10295,33 @@ bool OSD::maybe_override_options_for_qos()
   return false;
 }
 
+int OSD::mon_cmd_set_config(const std::string &key, const std::string &val)
+{
+  bufferlist inbl;
+  std::string outs;
+  C_SaferCond cond;
```

Comment:
might want to move cond & outs to where they are used (l. 10313)

Reply:
Done.